### ISSUE TITLE
MIDS Element - Altitude/depth

### ISSUE THREAD

**Issue by RBGE-Herbarium** [tdwg/mids#21]
**Summary**

MIDS information element: 

DwC term (latest, 2014-11-08): 

ABCD term name(3.0): 

Recommendations:

**Comment by PipBrewer** [comment]
Guess you mean depth e.g., in water bodies. Note that for archaeological practices, depths as part of archaeological stratigraphy is recorded. Not sure whether archaeology (e.g., faunal remains from the late Pleistocene) is in scope.

**Comment by smrgeoinfo** [comment]
Agree, property should not be required.  

Also, have to be clearer about what the number means:
depth-- vertical distance below some datum (sea surface, MSL, ground surface?), generally measured positive down. If you're dealing with boreholes, is the depth measure along the borehole trace (which might not be vertical), is it measured from ground surface or kelly bushing on the drill rig
altitude -- generally vertical distance above some datum (generally MSL), measured positive up
height-- generally vertical distance above  ground surface, measure positive up. 

so like horizontal coordinates, for a vertical coordinate to be interoperable, it needs to be bound with a coordinate reference system definition.

**Comment by tucotuco** [comment]
> so like horizontal coordinates, for a vertical coordinate to be interoperable, it needs to be bound with a coordinate reference system definition.

Agreed. See also [Darwin Core location terms](https://dwc.tdwg.org/terms/#location), where there are three sets of terms (around depth, elevation, and dsistanceAboveSurface) in order to be able to get a complex locations such as a bore at a depth in a water body whose surface is at an elevation.
